src/app/services/validation.service.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { FormControl, FormGroup } from '@angular/forms';
+import { ValidationService } from 'app/services/validation.service';
+
+describe('ValidationService', () => {
+  const service = new ValidationService();
+  const formControlName = 'value1';
+  const otherFormControlName = 'value2';
+  const formControlPlaceholder = 'Value 2';
+  let formGroup: FormGroup;
+  beforeEach(() => {
+    formGroup = new FormGroup({
+      value1: new FormControl(0, [
+        service.greaterThan(otherFormControlName, [formControlPlaceholder]),
+      ]),
+      value2: new FormControl(0),
+    });
+  });
+
+  it('should have greatherThan error when value1 is smaller', (done) => {
+    formGroup.get(formControlName).setValue(0);
+    formGroup.get(otherFormControlName).setValue(1);
+    formGroup.updateValueAndValidity();
+    expect(formGroup.valid).toBeFalsy();
+    expect(formGroup.get(formControlName).errors.greaterThan).toBeTruthy();
+    expect(formGroup.get(formControlName).errors.fields[0]).toBe(formControlPlaceholder);
+    done();
```

Comment:
`done` argument and calling `done()` is only really needed when you're testing async code. Not needed here.